doc/user/content/sql/drop-table.md

```diff
@@ -19,31 +19,85 @@ statements. `DROP TABLE` removes tables from your Materialize instance.
 
 Field | Use
 ------|-----
-**IF EXISTS** | Do not return an error if the named table does not exist.
-_table&lowbar;name_ | The name of the table to drop.
+`IF EXISTS`  | Do not return an error if the named table doesn't exist.
+_table_name_ | The name of the table to remove.
+`CASCADE` | Remove the table and its dependent objects.
+`RESTRICT`  | Don't remove the table if any non-index objects depend on it. _(Default.)_
 
 ## Examples
 
-### Dropping a table
-
-We can create a table `t` and verify that it was created with the following
-statements:
+### Remove a table with no dependent objects
+Create a table *t* and verify that it was created:
 
 ```sql
 CREATE TABLE t (a int, b text NOT NULL);
 
 SHOW TABLES;
+```
+```
 TABLES
 ------
 t
 ```
 
-We can then remove the table from the Materialize instance with a `DROP` statement.
+Remove the table:
+
+```sql
+DROP TABLE t;
+```
+### Remove a table with dependent objects
+
+Create a table *t*:
+
+```sql
+CREATE TABLE t (a int, b text NOT NULL);
+INSERT INTO t VALUES (1, 'yes'), (2, 'no'), (3, 'maybe');
+SELECT * FROM t;
+```
+```
+a |   b
+---+-------
+2 | no
+1 | yes
+3 | maybe
+(3 rows)
+```
+
+Create a materialized view from *t*:
+
+```sql
+CREATE MATERIALIZED VIEW t_view AS SELECT sum(a) AS sum FROM t;
+SHOW MATERIALIZED VIEWS;
+```
+```
+    name
+---------------
```

Comment:
this formatting looks off; the way that the dashes center under the column name doesn't do this in the output itself